src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,319 @@
+        <button onClick={captureMedia}>{'.media test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+** `.range` – Prompts the user to select a numeric range.
+<<core-frontend-ui-capture#range,API reference>>
++
+.View image and example.
+[%collapsible]
+====
+With `Koji.ui.capture.range({ initialValue : 50, min : 30, max : 70, step : 10 });` :
+
+image::ui-capture-range.png[UI / Capture .range control,50%,link="{imagesDir}/ui-capture-range.png"]
+
+.Example
+[source,javascript]
+----
+  const captureRange = async () => {
+    const capturedRange = await Koji.ui.capture.range({ initialValue : 50, min : 30, max : 70, step : 10 });
+    if (capturedRange) alert( 'You selected this value from the range: ' + capturedRange );
+  };
+  ...
+  return (
+    <div className="App">
+      <header className="App-header" style={{ backgroundColor }}>
+        <button onClick={captureRange}>{'.range test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+** `.select` – Prompts the user to select from a predefined list of options.
+<<core-frontend-ui-capture#select,API reference>>
++
+.View image and example.
+[%collapsible]
+====
+If the list has 1 to 4 options, the options are presented as radio buttons. +
+If the list has 5 or more options, the list is presented as a dropdown.
+
+image::ui-capture-select.png[UI / Capture .select control,50%,link="{imagesDir}/ui-capture-select.png"]
+
+.Example
+[source,javascript]
+----
+  const captureSelection = async () => {
+    const capturedSelection = await Koji.ui.capture.select(
+      {
+        placeholder : "Choose a mystery prize:",
+        options : [
+          { value : "prizeA", label : "Prize A" },
+          { value : "prizeB", label : "Prize B" },
+          { value : "prizeC", label : "Prize C" },
+          { value : "prizeD", label : "Prize D" },
+          { value : "prizeE", label : "Prize E" }
+        ]
+      }
+    );
+    if (capturedSelection) alert( 'You chose ' + capturedSelection );
+  };
+  ...
+  return (
+    <div className="App">
+      <header className="App-header" style={{ backgroundColor }}>
+        <button onClick={captureSelection}>{'.select test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+** `.video` – Prompts the user to select or upload a video file.
+<<core-frontend-ui-capture#video,API reference>>
++
+.View image and example.
+[%collapsible]
+====
+image::ui-capture-video.png[UI / Capture .video control,50%,link="{imagesDir}/ui-capture-video.png"]
+
+.Example
+[source,javascript]
+----
+  const captureVideo = async () => {
+    const capturedVideo = await Koji.ui.capture.video({ hideExtensions : false });
+    if (capturedVideo) alert( 'You chose/uploaded a video file: ' + capturedVideo );
+  }
+  ...
+  return (
+    <div className="App">
+      <header className="App-header" style={{ backgroundColor }}>
+        <button onClick={captureVideo}>{'.video test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+[TIP]
```

Comment:
A "tip" doesn't feel like the right solution. There's not enough info about why to use the different file upload methods. 

For example, you would use the specific types because they allow you set type-specific options (like putting a watermark on a video or optimizing an image). 

Media allows you to set type-specific options for several types of files with a single control (when that makes sense - like configuration a background that could be a static image or a video). 

File is best for uploading raw files (like a high-resolution image that someone is going to sell as a downloadable file, rather than display in the app). 

Upload is best for capturing raw data created in the app (or for providing a custom upload control). It's not even mentioned in the tip right now but there's potential for confusion with it and the other file upload methods.

Reply:
- Moved the "tip" content to a section.
- Added the reasons for each type-specific control.
- UI/Upload seems redundant with UI/Capture/.file. I added a link to UI/Upload instead.